news/_posts/2021-04-28-new-repository-service.md

```diff
@@ -11,8 +11,10 @@ meta:
   _edit_last: '4'
   _wpas_done_all: '1'
 ---
-We have spun up a new repository service at <a href="https://repos.spark-packages.org">https://repos.spark-packages.org</a> and it will be the new home for the artifacts on spark-packages.
+The spark-packages team has spun up a new repository service at <a href="https://repos.spark-packages.org">https://repos.spark-packages.org</a> and it will be the new home for the artifacts on spark-packages.
```

Comment:
Maybe link to spark-packages.org the first time you mention it so they know what "spark-packages" refers to.